apiserver/apiserver/apiserver.py

```diff
@@ -5,126 +5,157 @@
 
 import os
 import logging
-import flask
 import json
+import concurrent
+import uvloop
+import asyncio
+from aiohttp import web
```

Comment:
Do  you want `cchardet` as well? It is used to identify content encoding type when not provided explicitly https://docs.aiohttp.org/en/stable/client_reference.html. Minor, could do without.

As Jackie pointed out, `aiodns` is recommended. I'm still checking whether/where `aiodns` is used during normal routing operations.

Reply:
That's the client reference.  I'm using the server. I see no reference to to cchardet on the server side.  I don't think I need to do anything.

I included aiodns, although I don't technically think it's necessary here since this is "end of the line", and not resolving anything, but does seem like good practice.

I also realized there was a missing dependency from hail to apiserver (hail subproject runs apiserver tests).  This will now probably fail because the pr builder image needs to be updated to include the asyncio packages in the hail environment.  I don't want to duplicate that work so this will need to wait until @jigold's asyncio PR goes in.

Reply:
True, we're not issuing any requests, my mistake